### PR TITLE
Install AWS cli

### LINKED ISSUE #9: AWS cli is missing from boxbox
Based on conversation in https://github.com/renderedtext/boxbox/pull/8.

### PR DESCRIPTION
**Pull request by shiroyasha**
fixes https://github.com/renderedtext/boxbox/issues/9

### CODE REVIEW

**Comment by markoa** [comment]
This is required to have in `~/.zshrc`: `export PATH=~/.local/bin:$PATH`

File: tests.sh [hunk 1]
```diff
@@ -65,7 +65,12 @@ echo -e "\n[TEST] Docker compose version"
 docker-compose --version
 docker-compose --version | grep "docker-compose version 1.17"
 
+echo -e "\n[TEST] AWS cli version"
+export PATH=~/.local/bin:$PATH
+which aws
```
Comment (bmarkons) [inline tests.sh]: We can't do `grep "<version>"` because aws commands don't return any output. The output to the STDOUT is side effect :D

Comment (shiroyasha) [reply tests.sh]: @bmarkons Yeah, it seems that aws logs to stderr. You can redirect stderr to stdout to get the version:

```
aws --version 2>&1 | grep "1.12"
```

This passed on my computer.

Comment (bmarkons) [reply tests.sh]: Hah :D

Comment (shiroyasha) [reply tests.sh]: If you really want to hack, you can even swap stderr with stdout:

```
command 3>&1 1>&2 2>&3
```

Comment (bmarkons) [reply tests.sh]: lol 😆

**Comment by bmarkons** [comment]
fixes https://github.com/renderedtext/issues/issues/534#issuecomment-347447313

**Pull request merged**